resources/core/charts/console/templates/microfrontend.crd.yaml

```diff
@@ -19,8 +19,11 @@ spec:
       properties:
         spec:
           type: object
-          required: 
-          - "navigation"
+          required:
+          - "appName"
+          - "placement"
```

Comment:
is placement really required? even if it always has to be 'environment'?

Reply:
After talking to Philipp last week and Peter right now, we decided to remove placement from microfrontends completely.

Please change the examples to use ui.kyma.cx, since we did not continue to change the name to ui.kyma-project.io so far, due to external dependencies.